packages/nextjs-kit/src/__tests__/data/examplePostData.json

```diff
@@ -0,0 +1,8 @@
+[
+	{
+		"title": "Example Post with Image",
+		"date": "2022-08-04",
+		"featuredImage": "https://dev-decoupled-wordpress-qa.pantheonsite.i/wp-content/uploads/2022/08/pizza.jpeg",
+		"content": "<p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua.</p>\n \n \n \n \n \n <blockquote class=\"wp-block-quote\"><p>a Quote</p><cite>from QA</cite></blockquote>\n <figure class=\"wp-block-pullquote\" style=\"border-color:#0073a8\"><blockquote class=\"has-text-color has-dark-gray-color\"><p>a pull quoteeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeeee</p><cite>from qa</cite></blockquote></figure>\n \n \n \n <p>An<strong>other block</strong></p>\n"
```

Comment:
Can we use a more generic post without the 'a pull quoteeeeeeeeeeeeeeeeeeeeeeeee' stuff in it? :)